tests/unit/experimental/components/test_entity_relation_extractor.py

```diff
@@ -205,6 +210,14 @@ async def test_extractor_custom_prompt() -> None:
     llm.ainvoke.assert_called_once_with("this is my prompt")
 
 
+def test_fix_invalid_json_empty_result() -> None:
+    json_string = "invalid json"
+
+    with patch("json_repair.repair_json", return_value=""):
```

Comment:
Why do we need to patch the return value if the input json is invalid?

Reply:
When json_repair is unable to fix the JSON string, it returns `""`